bundler/doc/contributing/BUG_TRIAGE.md

```diff
@@ -33,11 +33,12 @@ If you can reproduce an issue, you're well on your way to fixing it. :)
 Everyone is welcome and encouraged to fix any open bug, improve an error message or add documentation. If you have a fix or an improvement to a ticket that you would like to contribute, we have a small guide to help:
 
   1. Discuss the fix on the existing issue. Coordinating with everyone else saves duplicate work and serves as a great way to get suggestions and ideas if you need any.
-  2. Base your commits on the correct branch. Bugfixes for 1.x versions of Bundler should be based on the matching 1-x-stable branch.
+  2. Base your commits on the correct branch. Bugfixes for 2.x versions of Bundler should be based on the matching 2-x-stable branch.
   3. Review the [pull request guide](../development/PULL_REQUESTS.md).
   4. Commit the code with at least one test covering your changes to a named branch in your fork.
-  5. Put a line in the [CHANGELOG](../../CHANGELOG.md) summarizing your changes under the next release under the “Bugfixes” heading.
-  6. Send us a [pull request](https://help.github.com/articles/using-pull-requests) from your bugfix branch.
+  5. Send us a [pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) from your bugfix branch.
+
+You do not have to update [CHANGELOG](../../CHANGELOG.md) in your PR. Release manager(s) will prepare it from the title of each PR.
```

Comment:
It's actually a release script that prepares the changelog from PR titles :)

Reply:
Do you want not to add this line? 😁

Reply:
Sorry, I should've suggested something. I'd go with.

```suggestion
You do not have to update [CHANGELOG](../../CHANGELOG.md) in your PR. Our release scripts will automatically prepare it from the title of each PR.
```